domain/relation/service/relation.go

```diff
@@ -185,8 +184,8 @@ func (s *Service) GetRelationID(ctx context.Context, relationUUID corerelation.U
 // GetRelationKey returns a key identifier for the given relation UUID.
 // The key describes the relation defined by endpoints in sorted order.
 // Note: See the state.relationKey() function.
```

Comment:
We should remember to get rid of this `Note` since state is going away.

Reply:
There is some work to do with the Key, this reference will be updated then.